app/femr/ui/views/admin/configure/manage.scala.html

```diff
@@ -13,13 +13,14 @@
             <tr>
                 <th>Feature</th>
                 <th>Toggle</th>
+                <th>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Description</th>
             </tr>
 
             @for((key, valyew) <- viewModel.getSettings) {
                 <tr>
                     <td class="name">@key</td>
                     <td class="isActive">
-                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} />
+                        <input type="checkbox" name="settings[]" value="@key" @if(valyew) { checked="checked"} >
                     </td>
```

Comment:
Hi Kratika -

FYI - You don't need to resubmit if you commit your change and push it to your fork on the same branch as this (it will update automatically in this pull request).

Thanks!!